frontend/src/util/apiUtil.js

```diff
@@ -1,21 +1,33 @@
-import Cookies from 'universal-cookie';
 import axios from 'axios';
 
-export const getToken = () => {
-    const cookies = new Cookies();
-    const token = cookies.get('token');
-    return token;
+export const captionRequestTypes = {
+    BY_MOMENT: 'BY_MOMENT',
+    BY_USER: 'BY_USER',
+    BY_MOMENT_AND_USER: 'BY_MOMENT_AND_USER',
 }
+export const captionFilters = ['Recent', 'Oldest', 'Top', 'Worst', 'Accepted', 'Rejected'];
+export const fetchCaptions = ({ token, type, filter, momentId, userId }) => {
```

Comment:
I would move this out into something like `getRequestTypeQuery` or something, and give it a similar implementation to `getFilterQuery`. That way this part would remain modular.